test/output_test_helper.cc

```diff
@@ -31,21 +31,21 @@ TestCaseList& GetTestCaseList(TestCaseID ID) {
 
 SubMap& GetSubstitutions() {
   // Don't use 'dec_re' from header because it may not yet be initialized.
-  static std::string dec_re = "[0-9]*[.]?[0-9]+([eE][-+][0-9]+)?";
+  static std::string dec_re2 = "[0-9]*[.]?[0-9]+([eE][-+][0-9]+)?";
```

Comment:
Otherwise ICC produces: declaration hides variable "<unnamed>::dec_re" (declared at line 67 of "../test/output_test.h"). I understood Dominic on #354 that it is better to fix the warning then suppress it.